indexer_certs_creator/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Wazuh Docker Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+FROM ubuntu:focal
+
+RUN apt-get update && apt-get install openssl -y
+
+WORKDIR /
+
+COPY config/unattended_installer.tar.gz /
+
+COPY config/entrypoint.sh /
+
+RUN tar -xzvf /unattended_installer.tar.gz
```

Comment:
The `wazuh-cert-tool.sh` must be finally downloaded from `https://packages.wazuh.com/X.Y/wazuh-cert-tool.sh` where `X.Y` is major-minor version

Reply:
Done